source/routing/defining-your-routes.md

```diff
@@ -301,13 +301,13 @@ Router.map(function() {
 ```
 
 Like all routes with a dynamic segment, you must provide a context when using a `{{link-to}}`
-or `transitionTo` to programatically enter this route.
+or `transitionToRoute` to programatically enter this route.
 
 ```app/routes/application.js
 export default Ember.Route.extend({
   actions: {
     error: function() {
-      this.transitionTo('catchall', 'application-error');
+      this.transitionToRoute('catchall', 'application-error');
```

Comment:
AFAIK `transitionToRoute()` is only available on controllers. Was this changed somewhere?